flink-filesystems/flink-fs-hadoop-shaded/pom.xml

```diff
@@ -207,29 +207,6 @@ under the License.
 									<include>*:*</include>
 								</includes>
 							</artifactSet>
-							<relocations>
```

Comment:
without relocations the README in this module is outdated.

Reply:
reverted all changes to this file